includes/media.php

```diff
@@ -246,6 +236,32 @@ function pods_attachment_import( $url, $post_parent = null, $featured = false )
 	return $attachment_id;
 }
 
+/**
+ * Resize an image on if it doesn't exist.
+ *
+ * @param int          $attachment_id Attachment ID
+ * @param string|array $size          Size to be generated
+ *
+ * @return boolean Image generation result
+ *
+ * @since 2.7.23
+ */
+function pods_maybe_image_resize( $attachment_id, $size ) {
+
+	if ( 'full' !== $size ) {
+		$full = wp_get_attachment_image_src( $attachment_id, 'full' );
+		if ( ! empty( $full[0] ) ) {
+			$src  = wp_get_attachment_image_src( $attachment_id, $size );
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 2 spaces

<sub>hash: 58dff66a0aba986b6c77af18cdb44dbd</sub>